packages/fetchai/skills/weather_station/dummy_weather_station_data.py

```diff
@@ -136,4 +139,4 @@ def generate(self) -> None:
 
 if __name__ == "__main__":
     a = Forecast()
-    a.generate()
+    a.generate(59)
```

Comment:
A bit arbitrary

Reply:
This is the number of rows in the current db.

Reply:
Maybe worth setting this as a constant for future reference?